MayssaraOmar/assignment3/Word search/src/test_dictionary.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TEST_DICTIONARY_H_
+#define TEST_DICTIONARY_H_
+
+#include "dictionary.h"
+
+class test_dictionary {
```

Comment:
Style: Generally classes are named in CamelCase, so I would suggest `DictionaryTests` for this class.